src/unwinder/arm64/unwinder_arm64.cpp

```diff
@@ -109,16 +109,37 @@ typedef struct _ARM64_VFP_STATE
 
 typedef struct _ARM64_UNWIND_PARAMS
 {
-    ULONG_PTR       ControlPc;
-    PULONG_PTR      LowLimit;
-    PULONG_PTR      HighLimit;
     PKNONVOLATILE_CONTEXT_POINTERS ContextPointers;
 } ARM64_UNWIND_PARAMS, *PARM64_UNWIND_PARAMS;
 
 #define UNWIND_PARAMS_SET_TRAP_FRAME(Params, Address, Size)
 
-#define UPDATE_CONTEXT_POINTERS(Params, RegisterNumber, Address)
-#define UPDATE_FP_CONTEXT_POINTERS(Params, RegisterNumber, Address)
+#define UPDATE_CONTEXT_POINTERS(Params, RegisterNumber, Address)                    \
+do {                                                                                \
+    if (ARGUMENT_PRESENT(Params)) {                                                 \
+        PKNONVOLATILE_CONTEXT_POINTERS ContextPointers = (Params)->ContextPointers; \
```

Comment:
I assume the fix to the build break is to use the type PT_KNONVOLATILE_CONTEXT_POINTERS instead of PKNONVOLATILE_CONTEXT_POINTERS. @janvorli this type is correctly shared between both the windows SDK and the PAL correct?